dashboard/app/views/levels/editors/fields/_music_level_data.html.haml

```diff
@@ -0,0 +1,6 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#level_data"}}
+  Level Data
+
+#level_data.in.collapse
+  = f.label :music_level_data, 'Level data JSON'
```

Comment:
should we add text here with the allowed values for the level data?

Reply:
That's a great idea, but I'm curious if we could address in a follow-up?  I've just added a bit of documentation for the format [here](https://github.com/code-dot-org/code-dot-org/blob/889b5f1bb5ad2c17d179a84b4f927cd06f671bf7/dashboard/app/models/levels/music.rb#L27-L34), and want to avoid having too many places to maintain as we still evolve it.  Right now, we'll have a very limited internal audience for these levels, and I see this PR as just the first step in building out a nicer editing experience.